vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -133,13 +133,13 @@ The following is a list of specifications for the bundle descriptor and its comp
     }
   ]
 ```
-::: tip  
- Entando uses the `healthCheckPath` to monitor the health of the microservice. A plugin in an Entando Bundle can use any technology, as long as it provides a health check service configured via the `healthCheckPath`. This path must be specified in the descriptor file and return an HTTP 200 or success status. This can be implemented by a Java service included with the Entando Blueprint in the Spring Boot application. You can also [use a Node.js service as shown here](https://github.com/entando-samples/ent-project-template-node-ms/blob/main/src/main/node/controller/health-controller.js). 
-:::
+#### Microservices Details
+ - It is recommended that REST APIs be added to microservices on Entando, following REST frameworks.
```

Comment:
this sentence needs some clarification; it reads like REST APIs should be added to MSs no matter what, not that if a dev wants to use a REST API they way they should go about that is by adding it to an MS. also, "to MS on Entando" reads like REST APIs should be added after-the-fact (as opposed to whenever - during or after MS creation)

following REST frameworks as in according to rules and guidelines imposed by REST frameworks? "according to REST frameworks" is more precise and has only one meaning, whereas "following" can read as either "in line with" or "after"

"it is recommended"... who is recommending this and why? that info is a nice-to-have

suggest something like: "To make use of REST APIs on Entando, add them to microservices in accordance with REST frameworks."